assets/js/pages/Event.tsx

```diff
@@ -72,16 +36,12 @@ const Event = () => {
   const socket = new Socket("/socket");
   socket.connect();
 
-  useEffect(() => {
-    getEventInfo(eventInfo, setEventInfo);
-  }, []);
-
   useEffect(() => {
     const alreadyJoined = eventChannel?.state === "joined";
     if (name && !alreadyJoined) {
-      const channel = socket.channel(`event:${eventInfo.link}`, {
+      const channel = socket.channel(`event:${getChannelId()}`, {
         name: name,
-        isModerator: eventInfo.isModerator,
+        isModerator: true,
```

Comment:
Are you sure that this should be always true?

Reply:
We don't have eventInfo state in Event component, isModerator will be added in the next pull request. Right now value is mocked.